pkg/engine/utils/exceptions.go

```diff
@@ -14,6 +16,7 @@ func MatchesException(
 	polexs []kyvernov2beta1.PolicyException,
 	policyContext engineapi.PolicyContext,
 	logger logr.Logger,
+	imageInfo apiutils.ImageInfo,
```

Comment:
The `imageInfo` would be used only for policies with verifyImage rules. I suggest adding a new function `ApplyVerifyImageExclusion()` that applies the exemption to images. And, this new function will be called from the `verifyImage` rules only. 

Here is the code for applying the `verifyImage` rules to resources: https://github.com/kyverno/kyverno/blob/main/pkg/engine/handlers/validation/validate_image.go

Reply:
Another solution: in the [verifyImage](https://github.com/kyverno/kyverno/blob/main/pkg/engine/handlers/validation/validate_image.go#L58) rules, if an exception does match the policy, then we can call `matchReferences()` and pass the `exception.spec.verifyImages.imageReferences` and check if the image matches with one of those which specified in the exception. If yes, we return `skip`.

This would be better than defining a new functionality in exceptions.go

Reply:
hi @MariamFahmy98 please check if this is what you are referring to 
```
if exception != nil && internal.MatchReferences(exception.Spec.VerifyImages.ImageReferences, image) {
	logger.V(4).Info("image matches exception references, skipping verification", "image", image)
	skippedImages = append(skippedImages, image)
	continue
	}
```